core/src/forwarding_stage.rs

```diff
@@ -444,7 +491,7 @@ impl<VoteClient: ForwardingClient, NonVoteClient: ForwardingClient>
                 .send_transactions_in_batch(non_vote_batch)
                 .is_err()
             {
-                self.metrics.votes_dropped_on_send += num_non_votes;
+                self.metrics.non_votes_dropped_on_send += num_non_votes;
```

Comment:
this was a bug in metric (wrong name), right?

Reply:
in the future, let's try to keep unrelated changes in their own PRs - it makes things like backporting easier. Is this metric in 2.2, should we backport this fix to the metric?

Reply:
I introduced this bug in the previous PR, so no backporting needed.